src/equations/numerical_fluxes.jl

```diff
@@ -157,9 +157,16 @@ Base.show(io::IO, d::DissipationLocalLaxFriedrichs) = print(io, "DissipationLoca
 
 Simple and fast estimate of the maximal wave speed of the Riemann problem with left and right states
 `u_ll, u_rr`, based only on the local wave speeds associated to `u_ll` and `u_rr`.
+
+For non-integer arguments `orientation_or_normal_direction`, `max_abs_speed_naive` returns
+`orientation_or_normal_direction * max_abs_speed_naive(u_ll, u_rr, 1, equations)`.
```

Comment:
That's not completely correct for multiple space dimensions, isn't it?

Reply:
Good point. I'll edit that, and the docstring is also missing an `abs`.